onnx/helper.py

```diff
@@ -290,7 +290,7 @@ def make_empty_tensor_value_info(name):  # type: (Text) -> ValueInfoProto
 def make_tensor_value_info(
         name,  # type: Text
         elem_type,  # type: TensorProto.DataType
-        shape,  # type: Optional[Sequence[int]]
+        shape,  # type: Optional[Sequence[TypeVar['dimension', Text, int]]]
```

Comment:
Would Sequnce[Union(Text,int)] be simpler? I don't understand TypeVar usage, so I am not sure. Just wondering.

Reply:
Yes I've switched to Union in latest commit